crates/outbound-redis/src/lib.rs

```diff
@@ -70,8 +71,12 @@ impl OutboundRedis {
 
 impl v2::Host for OutboundRedis {}
 
+// TODO: #[instrument(err)] is only reporting the outer error (if the guest traps), we want to mark
```

Comment:
This is an outbound call (from guest to host) - I'm not sure what "if the guest traps" can mean in this context?

Reply:
I mean to say that the outer error will cause the guest to trap. I could have worded that better.